pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -1057,6 +1065,36 @@ func (r *ReconcilePerconaServerMongoDB) reconcileMongos(cr *api.PerconaServerMon
 	return nil
 }
 
+func ensurePVCs(
+	ctx context.Context,
+	cl client.Client,
+	namespace string,
+	pvcs []corev1.PersistentVolumeClaim,
+) error {
+	for _, pvc := range pvcs {
+		// ignore pvc namespace
```

Comment:
```suggestion
		// Users configure the PVC template through CR.
		// We always override the namespace in case they specify a different one.
```

I had a hard time understanding this part. This comment could make it more clear. What do you think?